kong/plugins/session/handler.lua

```diff
@@ -2,26 +2,23 @@ local BasePlugin = require "kong.plugins.base_plugin"
 local access = require "kong.plugins.session.access"
 local session = require "kong.plugins.session.session"
 
+local kong = kong
 
 -- Grab pluginname from module name
 local plugin_name = ({...})[1]:match("^kong%.plugins%.([^%.]+)")
 
-local KongSessionHandler = BasePlugin:extend()
+local KongSessionHandler = {
+  PRIORITY = 1900,
+  VERSION = "1.0.0",
```

Comment:
It seems like the current version of this plugin in its rockspec is 1.0.2? https://github.com/Kong/kong-plugin-session/blob/master/kong-plugin-session-1.0.2-3.rockspec#L9

Reply:
I had changed this to match the rockspec but was told they were not the same thing. Still a bit confused on what versions to sync.
https://kongstrong.slack.com/archives/CEDRYUABS/p1545855566003000

Reply:
Hmmm I don't see any reason for not syncing the plugin's and the rockspec's version (afaik, all plugins have been doing so, and all Lua libraries that I know of follow a similar pattern). The revision number is different, and only applies to the rockspec itself.

Typically:

- Plugin/Library gets a version number: `"x.y.z"`
- The rockspec points to the `x.y.z` tag to refer to the version it needs to install
- The rockspec's version is `"x.y.z-n"`, where `n` is the revision number for the rockspec. It is common that a rockspec needs an update (fix/feature) that doesn't affect the plugin/library itself, and in such cases, the maintainer bumps the revision number to `n+1`.